osquery/core/posix/process.cpp

```diff
@@ -177,13 +177,21 @@ std::shared_ptr<PlatformProcess> PlatformProcess::launchPythonScript(
     const std::string& args) {
   std::shared_ptr<PlatformProcess> process;
   std::string argv;
-
-  if (!isPlatform(PlatformType::TYPE_FREEBSD)) {
-    argv = "/usr/local/osquery/bin/python " + args;
+  std::string osquery_path;
+
+  char const* tmp = getenv("OSQUERY_DEPS");
```

Comment:
Sure of course! Wasn't aware of it ;) Are you talking about process.h's `getEnvVar()`?

I agree about the unit test issue. I was wandering why python was hardcoded there if in the end the package can be installed on a system without `/usr/local/osquery`. I'm not sure I know the code base enough for a PR though.